live-examples/js-examples/typedarray-tostring.html

```diff
@@ -0,0 +1,7 @@
+<pre>
+<code id="static-js" class="language-js">var uint8 = new Uint8Array([10, 20, 30, 40, 50]);
+
+console.log(uint8.toString());
+// expected output: 10,20,30,40,50
+</code>
```

Comment:
Nit: It's unfortunate that this looks the same as `console.log(uint8);`. Is it worth calling something stringy on it, and logging that?

```
var uint8 = new Uint8Array([10, 20, 30, 40, 50]);

var uint8String = uint8.toString();

console.log(uint8String.startsWith("10"));
// expected output: true
```